src/pages/docs/runbooks/config-as-code-runbooks.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-11-24
+modDate: 2024-11-24
+title: Configuration as Code runbooks
+description: Details about using configuration as code with runbooks.
+navOrder: 20
+icon: fa-brands fa-git-alt
+---
+
+- cac runbooks allows you to version control your runbook process alongside your project code - just as you can already do with your cac deployment process, deployment settings and non-sensitive variables.
+- Cac runbooks is optional. On existing projects and non-VCS projects, you can continue using runbooks just as you always have.
+
```

Comment:
I wonder if it might be good to have some explanation of the benefits of using CaC runbooks here somewhere? It's kinda explained in this first dot point, and also further down in the differences section, leading with the "why should you use CaC runbooks" could potentially help to frame the rest of the doc.